.github/scripts/release.py

```diff
@@ -81,19 +86,20 @@ def make_package_index_file(pkg_index):
     with open(pkg_index_w_path, "w") as pkg_file:
         pkg_file.write(pkg_index_json_obj)
 
-def build_package_index_json(pkg_name, version):
-    package_index = get_latest_package_index_json()
+def build_package_index_json(pkg_name, version, repository):
+    # package_index = get_latest_package_index_json() # get online package index json
+    package_index = get_local_package_index_json() # get local package index template
```

Comment:
Can you explain a bit the reasons behind these changes?

I don´t think this would work... In the original implementation the existing .json file is downloaded and the latest release appended. 
In the new implementation the file from the template from package/ is used. How are the previous versions added to the file?